samples/nginx.conf

```diff
@@ -0,0 +1,13 @@
+server {
+    listen 80;
+
+    error_page 404 /404.html;
```

Comment:
```suggestion
    server_name deriv_api_bff_nginx
    error_page 404 /404.html;
```

Reply:
@shan-96  `server_name` is used for patten matching of http host name in request, here we don't want to limit this. if I test in container, I'll be accessing as name if docker container, if I testing on my laptop I'll be accessing this server as localhost.